lib/liquid/tags/include.rb

```diff
@@ -45,6 +45,7 @@ def parse(_tokens)
     end
 
     def render_to_output_buffer(context, output)
+      return output << options[:locale].t("errors.disabled.include") if context.registers['disabled_tags']&.disabled?('include')
```

Comment:
Should we add this to `Tag` so that all tags automatically get this check?

Reply:
Is there a way in ruby to be able to do this? My mind just goes to running super but that means I'd have to change all tags to be super at the start of this method?

I have ended up simplifying this a bit to 
```
return output << disabled_response if disabled?(context)
```